src/mixins/common-mixin.ts

```diff
@@ -1,27 +1,48 @@
 import { Component, Vue } from 'vue-property-decorator'
+import { mapState } from 'vuex'
 import { omit, isEqual } from 'lodash'
+import { EntityTypes } from '@/enums'
 
 /**
  * Mixin that provides some useful common utilities.
  */
-@Component({})
+@Component({
+  computed: {
+    ...mapState(['entityType'])
+  }
+})
 export default class CommonMixin extends Vue {
+  readonly entityType!: EntityTypes
+
+  /** Returns True if entity is a Benefit Company. */
+  isBComp (): boolean {
+    return (this.entityType === EntityTypes.BCOMP)
+  }
+
+  /** Returns True if entity is a Cooperative. */
+  isCoop (): boolean {
+    return (this.entityType === EntityTypes.COOP)
+  }
+
+  /** Returns True if entity is a Corporation. */
+  isCorp (): boolean {
+    return (this.entityType === EntityTypes.CORP)
+  }
+
```

Comment:
Good change 👍 Large refactoring tho, your brave!

Reply:
What can I say? I like clean code! Makes it easier to review, too.